vllm/entrypoints/openai/api_server.py

```diff
@@ -31,20 +31,13 @@
 from vllm.transformers_utils.tokenizer import get_tokenizer
 from vllm.utils import random_uuid
 
-try:
-    import fastchat
-    from fastchat.conversation import Conversation, SeparatorStyle
-    from fastchat.model.model_adapter import get_conversation_template
-    _fastchat_available = True
-except ImportError:
-    _fastchat_available = False
```

Comment:
Since this PR drops support for fastchat can you also take a quick search to see if there is any additional fastchat import anywhere else to remove it?

Reply:
Yes, there are no other references that I found.